.golangci.yaml

```diff
@@ -17,6 +17,15 @@ linters:
 linters-settings: # please keep this alphabetized
   staticcheck:
     go: "1.17"
-    checks: [ "all" ]
+    checks: [
+      "all",
+      "-S1*",   # Omit code simplifications for now.
+      "-ST1*",  # Mostly stylistic, redundant w/ golint
+      "-SA5011" # Possible nil pointer dereference
+    ]
+    ignore_pattern: [
```

Comment:
Is this a thing?  I don't see it in https://golangci-lint.run/usage/configuration/

Reply:
It is indeed not a thing, I will remove it.